x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/apm_jsbase/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "apm_jsbase",
+  "title": "APM: RUM Javascript",
+  "description": "Detect problematic spans and identify user agents that are potentially causing issues.",
+  "type": "APM data",
+  "logoFile": "logo.json",
+  "defaultIndexPattern": "apm-*",
+  "query": {
+    "bool": {
+      "filter": [
+        {
```

Comment:
We normally condense these clauses into a single line of JSON to improve readability.

Reply:
https://github.com/elastic/kibana/pull/53792/commits/f40d0a205dc8e20394e6900b6f98946d857a5b69